### PR TITLE
Add AFA

### PR DESCRIPTION
**Pull request by Tradecoin-Token**

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -2479,6 +2479,51 @@
         "osmosis-price:uosmo:649"
       ]
     },
+    {
+      "description": "Airdrop For All [AFA - New Name on Cosmos Ecosystem, A4A - Old Name on TurtleNetwork] is a token from turtleNetwork towards cosmos ecosystem.",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:2483]: Please do not modify the assetlist file--only zone assets

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -2983,6 +2983,13 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-01-18T15:42:00Z"
     },
+    {
+      "chain_name": "juno",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2987]: please relocate the asset to the bottom of the array

File: osmosis-1/osmosis.zone_assets.json [hunk 3]
```diff
@@ -2983,6 +2983,13 @@
...
+      "base_denom": "cw20:juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+      "path": "transfer/channel-169/cw20:juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+      "osmosis_verified": false,
+      "listing_date_time_utc": "2024-04-20T07:27:20Z"
+    },
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2992]: can you please include a comment property with the token name and symbol

**Comment by Tradecoin-Token** [comment]
Now Asset is registered at the Cosmos Chain Registry. https://github.com/cosmos/chain-registry/tree/master/juno/.
Please process it further.

**Comment by Tradecoin-Token** [comment]
Even AFA is merged into data,  where searching for AFA or the pool ID (1701) in the search bar doesn't show any results. Can you please check reasons and update it.

**Comment by JeremyParish69** [comment]
We hadn't run the automation yet, but we have now and I can see it in search.

**Pull request merged**